quizapp/src/main/java/billennium/quizapp/webConfig/WebConfig.java

```diff
@@ -0,0 +1,25 @@
+package billennium.quizapp.webConfig;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
+
+@Configuration
+public class WebConfig {
+
+    @Value("${allowedOrigin}")
+    private String allowedOrigin;
+
+    @Bean
+    public WebMvcConfigurer configurer() {
+        return new WebMvcConfigurer() {
```

Comment:
to bym po prostu zmienił by klasa WebConfig implementowała WebMvcConfigurer i bez @Bean 
Finalnie: 
@Configuration
@EnableWebMvc
public class WebConfig implements WebMvcConfigurer {
 
    @Override
    public void addCorsMappings(CorsRegistry registry) {
.....
    }
}

Reply:
true 👍 . daje dużą okejke

Reply:
PS> za implementowanie interfejsu daje okejke, jeżeli chodzi i @EnableWebMvc to napisalem komentarz wyzej